index.html

```diff
@@ -1053,6 +1053,17 @@ <h4>
     this specification.
   </p>
 
+  <section><h3>Feature support</h3>
+    <p>
+      Detecting if Web NFC is supported can be done by checking NDEFReader
+      and/or NDEFWriter objects.
+    </p>
+    <pre class="example">
+      if ('NDEFReader' in window) { /* ... Scan NDEF Tags */ }
+      if ('NDEFWriter' in window) { /* ... Write NDEF Tags */ }
+    </pre>
```

Comment:
Add that note that this doesn't necessarily mean that there is NFC hardware or that it is enabled system wide